api/src/main/java/io/quarkiverse/kafkastreamsprocessor/api/configuration/Configuration.java

```diff
@@ -50,18 +50,30 @@ public interface Configuration {
      */
     void setSourceValueSerde(Serde<?> serde);
 
+    /**
+     * @return The serde for values fed to the {@link Processor}
+     */
+    Serde<?> getSourceValueSerde();
```

Comment:
Why do you need getters here ?

Reply:
It's not strictly needed in terms of API, but it allows more advanced logic for config customization in user code.